src/PKSim.UI/Views/Individuals/ExpressionParametersView.cs

```diff
@@ -102,9 +102,9 @@ private void onCellMerge(CellMergeEventArgs e)
 
       protected virtual bool ShouldMergeCell(GridColumn column, TExpressionParameterDTO p1, TExpressionParameterDTO p2, bool representSameOrgan)
       {
-         //We only merge compartment values if they are empty (across organ) or if they represent the same organ
+         //We only merge compartment values if they represent the same organ
          if (Equals(column, _colCompartment.XtraColumn))
-            return string.IsNullOrEmpty(p1.CompartmentName) || representSameOrgan;
+            return  representSameOrgan;
```

Comment:
This was the wrong condition. We only merge in the compartment column if the values represent the same organ. Otherwise...we don't. I think this makes sense

Reply:
sounds reasonable :)